definitions.py

```diff
@@ -0,0 +1,124 @@
+class DoricFile:
+
+    """
+    Doric file structure names
+    """
+
+    class Group:
+        DATA_PROCESSED = "DataProcessed"
+
+    class Dataset:
+        IMAGE_STACK = "ImageStack"
+        TIME        = "Time"
+        ROI         = "ROI{0}"
+
+    class Attribute:
```

Comment:
I rearrange the attributes to match more the `hdf5definitions.h`. Could you make sure that the names are used well in the other files?